server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/GlobalOrdinalsStringTermsAggregator.java

```diff
@@ -87,6 +92,7 @@ public GlobalOrdinalsStringTermsAggregator(String name, AggregatorFactories fact
                                                boolean showTermDocCountError,
                                                Map<String, Object> metadata) throws IOException {
         super(name, factories, context, parent, order, format, bucketCountThresholds, collectionMode, showTermDocCountError, metadata);
+        this.resultStrategy = resultStrategy.apply(this);
```

Comment:
Nit: Since the syntax backing this is somewhat rare in aggs, maybe toss a quick comment here that we need the reference to `this` in order to construct the closure?  I figured it out, but did a double take when I first saw it.

Reply:
:+1: